src/map/navi.cpp

```diff
@@ -323,8 +323,8 @@ int32 map_type(const struct map_data * m) {
 		return std::find_if(m->navi.warps_outof.begin(), m->navi.warps_outof.end(), [&link](const navi_link* link2) {
```

Comment:
Actually I think one needs to be `into` and one `outof`, the idea is to see if one in_warp can't be reached by a out_warp